static/js/util.ts

```diff
@@ -114,22 +122,23 @@ export function robust_uri_decode(str) {
 // doesn't support the ECMAScript Internationalization API
 // Specification, do a dumb string comparison because
 // String.localeCompare is really slow.
-export function make_strcmp() {
+export function make_strcmp(): (x: string, y: string) => number {
     try {
         const collator = new Intl.Collator();
+        // eslint-disable-next-line @typescript-eslint/unbound-method
         return collator.compare;
```

Comment:
We should fix this rather than disabling the lint rule: `collator.compare.bind(collator)`.